tests/src/test/scala/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -630,218 +161,210 @@ class ApiGwTests
             rr.stdout should include regex (s"API Name:\\s+${testapiname}\n")
             rr.stdout should include regex (s"URL:\\s+")
             rr.stdout should include(testbasepath + testrelpath)
-        }
-        finally {
-            val finallydeleteActionResult = wsk.action.delete(name = actionName, expectedExitCode = DONTCARE_EXIT)
-            val deleteresult = apiDelete(basepathOrApiName = testbasepath)
+        } finally {
+            wsk.action.delete(name = actionName, expectedExitCode = DONTCARE_EXIT)
+            wsk.api.delete(basepathOrApiName = testbasepath)
         }
     }
 
     it should "verify successful creation and deletion of a new API" in {
         val testName = "CLI_APIGWTEST1"
-        val testbasepath = "/"+testName+"_bp"
+        val testbasepath = "/" + testName + "_bp"
```

Comment:
Going to leave these as concat expressions. It would have to be ${...}_bp which makes + easier to read imo.